packages/11ty/_includes/components/figure/youtube/print.js

```diff
@@ -0,0 +1,27 @@
+const { html } = require('~lib/common-tags')
+const chalkFactory = require('~lib/chalk')
+
+/**
+ * Renders an image instead of a Youtube video player
+ *
+ * @param      {Object}  eleventyConfig  eleventy configuration
+ * @param      {Object}  figure          The figure
+ * @return     {String}  An HTML
+ */
+module.exports = function(eleventyConfig) {
+  const figurecaption = eleventyConfig.getFilter('figurecaption')
+  const figurelabel = eleventyConfig.getFilter('figurelabel')
+
+  const { figureLabelLocation } = eleventyConfig.globalData.config.params
+
+  return function({ aspect_ratio: aspectRatio, caption, credit, id, label, src}) {
+
+    return html`
+      <div class="q-figure__media-wrapper--${ aspectRatio || 'widescreen' }">
+        <img src="${src}" alt="" />
```

Comment:
It may be safe to omit the `alt` attribute for print output?
```suggestion
        <img src="${src}" />
```